fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/AgilityContextTest.java

```diff
@@ -704,6 +706,26 @@ void testAutoCommitVersionStampInnerSleepUseApply() {
         }
     }
 
+
+    @Test
+    void luceneTransactionPriorityVerificationTest() {
```

Comment:
It seems like it would be good to parameterize this test with the `userPriority` being set to `DEFAULT` and `BATCH` and the `agilePriority` being set to inherity or use `BATCH`.

Reply:
I've simplified a bit. Hope it's good enough.